### ISSUE TITLE
Support reloading

### ISSUE THREAD

**Issue by NoMathExpectation** [xxfast/KStore#124]
Hi, currently stores don't have a way to reload the data if I manually modified the corresponding file (at least I didn't find one). I think it would be great if there is a method to manually reload the store.

**Comment by xxfast** [comment]
Hi. Do you mean with something like the [FileObserver](https://developer.android.com/reference/android/os/FileObserver)?

**Comment by NoMathExpectation** [comment]
> Hi. Do you mean with something like the [FileObserver](https://developer.android.com/reference/android/os/FileObserver)?

Yes, sure. If the store can be reloaded automatically that would surely be better.
